lib/ethereum/cw_ethereum.dart

```diff
@@ -157,4 +168,24 @@ class CWEthereum extends Ethereum {
   }
 
   String getTokenAddress(CryptoCurrency asset) => (asset as Erc20Token).contractAddress;
+
+  @override
+  void setLedger(WalletBase wallet, Ledger ledger, LedgerDevice device) {
+    ((wallet as EVMChainWallet).evmChainPrivateKey as EvmLedgerCredentials).setLedger(
+        ledger,
+        device.connectionType == ConnectionType.usb ? device : null,
```

Comment:
there is another comment questioning the necessity of this parameter, but if it's indeed necessary, why are we only passing it in the case of USB? can you elaborate this flow

Reply:
This is because `ledger.devices` only lists ledger-ble devices and not usb-devices. So we have to store the USB-connection differently.